### PR TITLE
Fix preference list scroll issue

### PR DESCRIPTION
**Pull request by DenisFrezzato**
This PR fixes https://github.com/teamdigitale/italia-app/issues/512.

preference-list

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a Pivotal story at the beginning of the PR title (see below).
  </td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/components/PreferenceItem.tsx [hunk 1]
```diff
@@ -29,11 +28,11 @@ type Props = ValueProps | ActionProps;
  */
 class PreferenceItem extends React.Component<Props> {
   public render() {
+    // FIXME: this brokes the TS type matching and types refinement.
```
Comment (cloudify) [inline ts/components/PreferenceItem.tsx]: what does it mean?

Comment (DenisFrezzato) [reply ts/components/PreferenceItem.tsx]: My fault, I did a wrong minor refactoring.

File: ts/screens/preferences/PreferencesScreen.tsx [hunk 2]
```diff
@@ -88,34 +88,43 @@ class PreferencesScreen extends React.Component<Props> {
         subtitle={I18n.t("preferences.subtitle")}
         contextualHelp={contextualHelp}
       >
-        <Content>
+        <Content noPadded={true}>
```
Comment (cloudify) [inline ts/screens/preferences/PreferencesScreen.tsx]: is this line the only thing that needs to be changed for this fix? everything else it seems refactoring the existing code?

Comment (DenisFrezzato) [reply ts/screens/preferences/PreferencesScreen.tsx]: No, I also had to move the `<ListItem>` out from the `<PreferenceItem>` to fix the style.

**Pull request merged**